README.md

```diff
@@ -1,2 +1,13 @@
 # Using-Bootstrap
 A solo project where I am tasked with creating a replica of the newsweek.com website using the Bootstrap framework
+
+# (- PROJECT: Using Bootstrap - )
+A solo project where I am tasked with creating a replica of the newsweek.com website using the Bootstrap framework. You can find the original project specification at: https://www.theodinproject.com/courses/html5-and-css3/lessons/using-bootstrap
+
+## Authors: 
+### Aaron
+Aaron's Github Profile: https://github.com/Aaron-RN
+
+
+## Github Link
+Github Link: https://github.com/Aaron-RN/Using-Bootstrap
```

Comment:
#### `OPTIONAL`

Can you add screenshots different breakpoint of the responsive [clone](https://raw.githack.com/Aaron-RN/Building-With-Responsive-Design/82df751cb251b9f875b5f4d04067da4119c3a56f/navbar.html). I think that would give this `README` a more descriptive and professional look. :blush: